amaranth/vendor/xilinx.py

```diff
@@ -411,6 +429,89 @@ def _part(self):
         """
     ]
 
+    # Yosys NextPNR prjxray templates
+
+    @property
+    def _xray_part_map(self):
```

Comment:
This could be called just `_xray_part`, since it's not returning a dictionary.

Reply:
Done